docs/getting-started-guides/dcos.md

```diff
@@ -0,0 +1,171 @@
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/getting-started-guides/dcos.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+Getting started with Kubernetes on DCOS
+----------------------------------------
+
+**Table of Contents**
+
+- [About Kubernetes on DCOS](#about-kubernetes-on-dcos)
+- [Prerequisites](#prerequisites)
+- [Install Kubernetes](#install-kubernetes)
+- [Operations](#operations)
+- [Uninstall Kubernetes](#uninstall-kubernetes)
+
+
+## About Kubernetes on DCOS
+
+The [Datacenter Operating System (DCOS)](https://docs.mesosphere.com/) is a collection of tools for managing distributed systems deployed on a cluster of machines (virtual or bare metal).
+
+Since Kubernetes can be configured to schedule through Mesos, the cluster kernel at the heart of DCOS, it's a natural fit to deploy Kubernetes on DCOS.
+
+This involves packaging the Kubernetes core components to run in containers and creating a deployment manifest so that those core components can be deployed with and kept alive by Marathon, the DCOS init system. Since this packaging is DCOS-specific, its source is in a different repo ([kubernetes-mesos](https://github.com/mesosphere/kubernetes-mesos)). But you don't have to built it yourself, you can use the packaged releases in the [Mesosphere Multiverse](https://github.com/mesosphere/multiverse) DCOS package repository.
+
+This tutorial will walk you through setting up Kubernetes on a DCOS cluster and starting your first pod with an nginx webserver.
+
+**NOTE:** There are [known issues with the current implementation](../../contrib/mesos/docs/issues.md).
+If you have problems completing the steps below, please [file an issue against the kubernetes-mesos project](https://github.com/mesosphere/kubernetes-mesos/issues).
+
+Further information about Kubernetes on Mesos is available in the [contrib directory](../../contrib/mesos/README.md).
+
+
+## Prerequisites
+
+- A running [DCOS cluster](https://mesosphere.com/product/)
+  - DCOS Community Edition is currently available on AWS (GCE & Azure coming soon).
+  - DCOS Enterprise Edition can be deployed on any virtual machines or bare metal. Contact sales@mesosphere.com for more info and to set up an engagement.
+- [DCOS CLI](https://docs.mesosphere.com/install/cli/) installed locally
+
+
+## Install Kubernetes
+
+1. Configure and validate the Mesosphere Multiverse as a package source repository
+
+  ```
+  $ dcos config prepend package.sources https://github.com/mesosphere/multiverse/archive/version-1.x.zip
+  $ dcos package update --validate
+  ```
+
+2. Install Etcd
+
+  In order to survive Kubernetes core component process and/or container loss, an external HA Etcd cluster is required.
```

Comment:
Probably more wrong because I only explain there's a local one later... I do want to explaint he distinction. Got better word preference?